bumble/smp.py

```diff
@@ -886,6 +887,15 @@ def distribute_keys(self):
             csrk = bytes(16)  # FIXME: testing
             if self.initiator_key_distribution & SMP_SIGN_KEY_DISTRIBUTION_FLAG:
                 self.send_command(SMP_Signing_Information_Command(signature_key=csrk))
+            
+            # CTKD, calculate BR/EDR link key
+            if self.initiator_key_distribution & SMP_LINK_KEY_DISTRIBUTION_FLAG:
+                ilk = crypto.h7(
+                    salt=bytes.fromhex(
+                        '00000000000000000000000000000000746D7031'),
```

Comment:
Done(=> crypto.H7_LEBR_SALT)

Reply:
Would be better to put this constant definition in smp.py because crypto.py is just a general crypto primitive toolbox (used by SMP, but not necessarily just SMP), whereas this constant is only relevant as part of the SMP spec.

Reply:
Done